CryptKeeperZK/crypt-keeper-extension
# fix: demo launch

## Explanation

Currently, it's not possible to demonstrate features of CK extension.
This is a first part which allows to launch demo website and get identity.
Next parts will include proof generation for Semaphore and RLN.

Details are bellow:

- [x] Add fixes for demo react app
- [x] Replace axios with native fetch
- [x] Minor prettier/lint fixes
- [x] Change proof generation interface

## More Information

See: #89

## Screenshots/Screencaps

N/A

## Manual Testing Steps

1. Just go through readme steps
2. Open demo page and connect with CK extension
3. You will see your active identity on the page

## Pre-Merge Checklist


### + If there are functional changes:


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
